README.md

```diff
@@ -1 +1,188 @@
-# Portfolio_Project
+<a name="Portfolio Project"></a>
+
+
+<div align="center">
+ 
+   
+
+
+  <h3><b>My Portfolio Project</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [My Portfolio] <a name="Basic HTML and css Page"></a>
+
+> This is basic my portfolio page I build it with HTML and CSS
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. Linters
+2. HTML
+3. CSS
```

Comment:
- [OPTIONAL]
These three highlighted parts are not needed as they currently look like repetition, the tech stack covers completely and is descriptive enough to tell anyone that these were the technologies used to build the application :+1: